core/server/api/v2/utils/serializers/output/utils/clean.js

```diff
@@ -37,6 +71,20 @@ const post = (attrs) => {
     delete attrs.status;
     delete attrs.visibility;
 
+    // We are standardising on returning null from the Content API for any empty values
+    if (attrs.twitter_title === '') {
+        attrs.twitter_title = null;
+    }
+    if (attrs.twitter_description === '') {
+        attrs.twitter_description = null;
+    }
+    if (attrs.meta_title === '') {
+        attrs.meta_title = null;
+    }
+    if (attrs.meta_description === '') {
+        attrs.meta_description = null;
+    }
+
```

Comment:
Was not 💯 if this is the right file for this cleanup or make a new one for these changes, as so far it only contained values to be deleted. 🤔 @gargol Let me know what you think 😄

Reply:
Hm.. I'm always suspicious on why these values are `''` in the first place? :thinking:

Reply:
because we have no control over what a client sends to us

Reply:
@kirrg001 has made a comment that supports the suspicion about `''` values - https://github.com/TryGhost/Ghost/pull/10378#issuecomment-454351703 . This code could be added here too keep responses consistent, but would lean towards normalizing our inputs than formatting our outputs.

Reply:
@gargol Yeah I like normalizing inputs over formatting outputs as well, though I think it will still need to be coupled with formatting output for now until we do a migration for existing inputs. Minor concern I had for model layer level changes was if it affects `v0.1` in a way we don't want to.